### ISSUE TITLE
Browser authentication process not prompted in Fedora 34

### ISSUE THREAD

**Issue by dgasull** [oracle/netsuite-suitecloud-sdk#467]
### Your environment
**OS** : Fedora 34
**Node.js version** : v14.17.6
**suitecloud-cli version** : 1.3.0
**Terminal/CMD tool** : Bash
**Browser**: Firefox 92.0
**WebStorm**: 2021.2.2 
**SuiteCloud plugin IDE**: 2021.2.1

### Describe the bug
Cannot open browser authentication tab while trying to setup account. `Continue the authentication process in prompted browser` message appears but no browser tab/window is prompted. 

### To Reproduce
1. Run `suitecloud account:setup -d` 
2. Introduce your domain
3. Select `Browser-based authentication`
4. Introduce authentication ID 
5. No browser authentication window/tab is opened
![Screenshot from 2021-09-28 11-38-09](https://user-images.githubusercontent.com/38685890/135063082-1d944462-0002-4a21-9a09-be0fe454a577.png)

Same happens for browser authentication for `WebStorm SuiteCloud IDE plugin` :
1. Open Webstorm 
2. Go to File - Settings - Tools - NetSuite
3. Go to Account Management and click on `Add via browser authentication`
4. No browser authentication window/tab is opened
![Screenshot from 2021-09-28 11-40-25](https://user-images.githubusercontent.com/38685890/135063592-afc86c30-6b0f-400e-9dcd-28b5ab388369.png)

### Expected Behavior
A tab/window in browser should open

### Additional context 
Firefox 92 is default browser 
Pop-up blocking was disabled

**Comment by alii-gator** [comment]
@dgasull Yes we are aware of the problem with Fedora 34, we are working on it.

**Comment by irrgit** [comment]
1. The process to open a browser tab completes after a while and I am prompted to login in NetSuite. The issue is that the generated URL trying to callback to http://localhost:52300/tba?oauth_token= fails since the connection is most likely closed by then.


```2021-10-05 14:58:48 - acp-test
The authentication process has encountered an error. Close any tabs you have used to authenticate other NetSuite roles, and try again.
2021-10-05 15:00:25 - acp-test
The creation of a new authentication ID (authID) has been canceled.
2021-10-05 15:01:03 - acp-test
The authentication process has encountered an error. Close any tabs you have used to authenticate other NetSuite roles, and try again.
2021-10-05 15:08:33 - acp-test
The connection has timed out.
```
2. Clicking "Cancel" after starting the request immediately opens up the tab in the browser

**Comment by VictorJuez** [comment]
Now it works for me :upside_down_face:
